index.html

```diff
@@ -20,7 +19,8 @@
         </a>
       </div>
       <nav>
-        <div id="hamburger-button"><i class="fa fa-bars"></i></div>
+        <label for="nav-check" id="hamburger-button"><i class="fa fa-bars"></i></label>
+        <input id="nav-check" name="nav-check" type="checkbox">
```

Comment:
why do we need this input?

Reply:
I used this input as a check to open and close mobile navigation bar.
I used css only for hamburger menu